api/config/config_test.json

```diff
@@ -140,6 +140,7 @@
     "usersPerSynchronizeAccessTask": 50
   },
   "egressAlertRemediationPolicy": {
+    "enableJiraTicketing": false,
```

Comment:
Do we need to add this into other envs, would it cause NPE in those envs?

Reply:
I've made it legal to have a null `egressAlertRemediationPolicy` in other environments for now, so this is fine (they don't specify this config at all).